DataFormats/CTPPSReco/interface/CTPPSFastTrack.h

```diff
@@ -0,0 +1,98 @@
+#ifndef CTPPSFastTrack_H
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h